browser/policy/brave_configuration_policy_handler_list_factory.cc

```diff
@@ -0,0 +1,23 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/policy/brave_configuration_policy_handler_list_factory.h"
+
+#include "brave/browser/policy/brave_https_upgrade_policy_handler.h"
+#include "brave/browser/policy/brave_simple_policy_map.h"
+#include "components/policy/core/browser/configuration_policy_handler_list.h"
+
+namespace policy {
+
+void AddBraveHandlersToList(ConfigurationPolicyHandlerList& handlers) {
```

Comment:
I don't understand why you put this here when there is already a chromium_src override. What is the point of splitting this out from BuildHandlerList?

Reply:
Thanks for comment! Agreed, so I removed additional files and now add handlers directly in chromium_src override